Bobcat/Terminal.h

```diff
@@ -27,9 +27,8 @@ struct Terminal : TerminalCtrl {
     void        Restart();
     void        Reset();
     
-    bool        ShouldExit() const;
-    bool        ShouldKeep() const;

```

Comment:
I ave noticed that `ShouldKeep` is never called, so I deleted it. Let me know if you'd like to keep it for some reason.